src/plotloader/ScatterPlot.js

```diff
@@ -3,39 +3,77 @@ import PropTypes from 'prop-types'
 import ReactHighcharts from 'react-highcharts'
 import HighchartsExporting from 'highcharts/modules/exporting'
 import HighchartsBoost from 'highcharts/modules/boost'
-
+import HighchartsHeatmap from 'highcharts/modules/heatmap.js'
+import HighchartsMap from 'highcharts/modules/map'
 import deepmerge from 'deepmerge'
 
+import HeatmapLegend from './HeatmapLegend'
 import SeriesPropTypes from './SeriesPropTypes'
+import Yaxispanning from './Yaxispanning'
+
 
-const Highcharts = ReactHighcharts.Highcharts
+const Highcharts = ReactHighcharts.Highcharts;
 // Only apply modules if Highcharts isn’t a *good* mock -- Boost/Exporting can break tests
 // if (Highcharts.getOptions()) {
-  HighchartsExporting(Highcharts)
-  HighchartsBoost(Highcharts)
-// }
+async function addModules(){
+
+  await HighchartsExporting(Highcharts);
+  await HighchartsBoost(Highcharts);
+  await HighchartsMap(Highcharts);
+  await HighchartsHeatmap(Highcharts);
+  await HeatmapLegend(Highcharts);
+  await Yaxispanning(Highcharts);
+}
+
+addModules();
```

Comment:
Why are these modules wrapped in an asynchronous function?

Reply:
I added a comment in [Pivotal](https://www.pivotaltracker.com/story/show/157878166/comments/193178664) about this issue before. Please have a look into it.